src/server/dev.ts

```diff
@@ -2,15 +2,44 @@ import compression from 'compression';
 import history from 'connect-history-api-fallback';
 import express from 'express';
 import http from 'http';
+import minimist from 'minimist';
 import favicon from 'serve-favicon';
+import sio from 'socket.io';
 import webpack from 'webpack';
 import webpackDevMiddleware from 'webpack-dev-middleware';
 
 import webpackConfig from '../../webpack.config';
 
+import { UDPServer } from './network/network';
+
+const args = minimist(process.argv.slice(2));
+const redTeamAddress = args.team_a || '239.226.152.162';
+const blueTeamAddress = args.team_b || '239.226.152.163';
+
 const compiler = webpack(webpackConfig);
 const app = express();
 const server = http.createServer(app);
+const sioNetwork = sio(server);
+
+// Start listening to the multicast groups
+const udpServer = UDPServer.of(redTeamAddress, blueTeamAddress);
```

Comment:
I feel like having multiple servers listening on an address/port each makes more sense. Also, there are no ports going in here which is super suspicious